src/BlockTypes/CatalogSorting.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * CatalogSorting class.
+ */
+class CatalogSorting extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'catalog-sorting';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string | void Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		ob_start();
+		woocommerce_catalog_ordering();
+		$catalog_sorting = ob_get_clean();
+
+		if ( ! $catalog_sorting ) {
+			return;
+		}
+
+		$classname          = isset( $attributes['className'] ) ? $attributes['className'] : '';
+		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
+		return sprintf(
+			'<div class="wc-block-catalog-sorting %s %s" style="%s">%s</div>',
```

Comment:
WordPress CS used to throw a warning/error about `Multiple placeholders should be ordered.` Not sure why it doesn't anymore. But I still think we should apply that practice. We should also add the `woocommerce` class like we do to `Store Notices` and `Product Results Count` to inherit the style from WooCommerce.

```suggestion
			'<div class="woocommerce wc-block-catalog-sorting %1$s %2$s" style="%3$s">%4$s</div>',
```